### PR TITLE
upgrade dependencies

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-upgrade-deps](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-upgrade-deps)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-upgrade-deps](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-upgrade-deps)

File: package.json [hunk 1]
```diff
@@ -69,9 +69,9 @@
     "@skyux/lookup": "4.2.0",
     "@skyux/modals": "4.3.1",
     "@skyux/omnibar-interop": "4.0.0",
-    "@skyux/popovers": "4.0.0",
+    "@skyux/popovers": "4.1.0",
     "@skyux/router": "4.0.0",
-    "@skyux/tabs": "4.1.1",
+    "@skyux/tabs": "4.2.0",
     "@skyux/theme": "4.4.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json:75]: Theme is at `4.7.0` now, FYI.

**Pull request merged**